.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
Hi @kaungkhant0307, 
According to this requirement, 

![image](https://github.com/kaungkhant0307/hellomicroverse/assets/113283174/4db4f1bb-068a-4d25-936a-ef6406d1f76e)

Kindly add `test.md` in the `.gitignore` file.